pip/resolve.py

```diff
@@ -149,7 +163,7 @@ def resolve(self, requirement_set):
             requirement_set.unnamed_requirements +
             requirement_set.requirements.values()
         )
-        require_hashes = (
```

Comment:
This is a slightly cleaner approach than passing around `require_hashes`... This probably the blink and you miss it change I think I've made.